revisions/F0/src/constants/charmaps/dialog_codepoints.asm

```diff
@@ -142,3 +144,5 @@ charmap "<ask>", $fe
 charmap "@", $ff
 
 POPC
+
+SETCHARMAP DialogCharmap
```

Comment:
I'm a bit worried by the need to auto-set the dialog charmap when including those file. Does it solve any specific issue that appeared in this PR? What if this line is removed?

Reply:
this line comes from [`main.asm`](https://github.com/zladx/LADX-Disassembly/blob/5e4ed8942b3617087bcc1d321613cdcdff369d7c/src/constants/charmaps/main.asm), which is one line long, and overridden by a [`J0 version`](https://github.com/zladx/LADX-Disassembly/blob/5e4ed8942b3617087bcc1d321613cdcdff369d7c/revisions/J0/src/constants/charmaps/main.asm) that's no longer needed. I assumed that it was only in a separate file in order to be overridden. I may have been wrong!

Reply:
Oh, right. Hmm, I'm still unsure about "including constant files that actually trigger some global behavior", but in the end I just added the `SETCHARMAP DialogCharmap` at the top of `src/header.asm`, and call it a day. I think it makes sense until we have something better.